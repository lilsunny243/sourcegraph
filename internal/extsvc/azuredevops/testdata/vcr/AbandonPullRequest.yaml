---
version: 1
interactions:
- request:
    body: '{"status":"abandoned","title":null,"description":null,"mergeOptions":null,"lastMergeSourceCommit":null,"targetRefName":null,"completionOptions":{"mergeStrategy":"","deleteSourceBranch":false,"mergeCommitMessage":""}}'
    form: {}
    headers:
      Content-Type:
      - application/json
    url: https://dev.azure.com/sgtestazure/sgtestazure/_apis/git/repositories/sgtestazure/pullrequests/12?api-version=7.0
    method: PATCH
  response:
    body: '{"repository":{"id":"c4d186ef-18a6-4de4-a610-aa9ebd4e1faa","name":"sgtestazure","url":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa","project":{"id":"dc493f7d-0b57-4de2-a59b-3f74ff3ea334","name":"sgtestazure","url":"https://dev.azure.com/sgtestazure/_apis/projects/dc493f7d-0b57-4de2-a59b-3f74ff3ea334","state":"wellFormed","revision":11,"visibility":"private","lastUpdateTime":"2023-01-20T19:22:47.403Z"},"size":3688630,"remoteUrl":"https://sgtestazure@dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazure","sshUrl":"git@ssh.dev.azure.com:v3/sgtestazure/sgtestazure/sgtestazure","webUrl":"https://dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazure","isDisabled":false,"isInMaintenance":false},"pullRequestId":12,"codeReviewId":12,"status":"abandoned","createdBy":{"displayName":"idan.varsano@sourcegraph.com","url":"https://spsprodcca1.vssps.visualstudio.com/Ab768986f-7304-43dd-be74-f37e30f66c38/_apis/Identities/473dec3e-03d7-6147-b106-0b0a7f766a92","_links":{"avatar":{"href":"https://dev.azure.com/sgtestazure/_apis/GraphProfile/MemberAvatars/aad.NDczZGVjM2UtMDNkNy03MTQ3LWIxMDYtMGIwYTdmNzY2YTky"}},"id":"473dec3e-03d7-6147-b106-0b0a7f766a92","uniqueName":"idan.varsano@sourcegraph.com","imageUrl":"https://dev.azure.com/sgtestazure/_api/_common/identityImage?id=473dec3e-03d7-6147-b106-0b0a7f766a92","descriptor":"aad.NDczZGVjM2UtMDNkNy03MTQ3LWIxMDYtMGIwYTdmNzY2YTky"},"creationDate":"2023-02-17T02:21:23.2995071Z","closedDate":"2023-02-21T17:21:20.9037353Z","title":"Make
      it clearer that \"command\" steps are executed on local machine","description":"Make
      it clearer that \"command\" steps are executed on local machine","sourceRefName":"refs/heads/action-docs","targetRefName":"refs/heads/master","isDraft":false,"mergeId":"168fc5d7-c830-45a0-b8c4-f69e16faa67a","lastMergeSourceCommit":{"commitId":"43f23696277d8c412aa5da481009a5ec8302ded4","url":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/commits/43f23696277d8c412aa5da481009a5ec8302ded4"},"lastMergeTargetCommit":{"commitId":"12fba599d6b6ba7de2d4c5194cea58090201e4e1","url":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/commits/12fba599d6b6ba7de2d4c5194cea58090201e4e1"},"reviewers":[],"url":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/pullRequests/12","_links":{"self":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/pullRequests/12"},"repository":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa"},"workItems":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/pullRequests/12/workitems"},"sourceBranch":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/refs/heads/action-docs"},"targetBranch":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/refs/heads/master"},"statuses":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/pullRequests/12/statuses"},"sourceCommit":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/commits/43f23696277d8c412aa5da481009a5ec8302ded4"},"targetCommit":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/commits/12fba599d6b6ba7de2d4c5194cea58090201e4e1"},"createdBy":{"href":"https://spsprodcca1.vssps.visualstudio.com/Ab768986f-7304-43dd-be74-f37e30f66c38/_apis/Identities/473dec3e-03d7-6147-b106-0b0a7f766a92"},"iterations":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/pullRequests/12/iterations"}},"completionOptions":{"mergeCommitMessage":"","mergeStrategy":"noFastForward"},"supportsIterations":true,"artifactId":"vstfs:///Git/PullRequestId/dc493f7d-0b57-4de2-a59b-3f74ff3ea334%2fc4d186ef-18a6-4de4-a610-aa9ebd4e1faa%2f12"}'
    headers:
      Access-Control-Expose-Headers:
      - Request-Context
      Activityid:
      - 133b8869-a401-49b1-8741-3817dc116205
      Cache-Control:
      - no-cache, no-store, must-revalidate
      Content-Type:
      - application/json; charset=utf-8; api-version=7.0
      Date:
      - Tue, 21 Feb 2023 17:21:20 GMT
      Expires:
      - "-1"
      P3p:
      - CP="CAO DSP COR ADMa DEV CONo TELo CUR PSA PSD TAI IVDo OUR SAMi BUS DEM NAV
        STA UNI COM INT PHY ONL FIN PUR LOC CNT"
      Pragma:
      - no-cache
      Request-Context:
      - appId=cid-v1:72d31d95-1757-44f5-b910-a46611808454
      Strict-Transport-Security:
      - max-age=31536000; includeSubDomains
      Vary:
      - Accept-Encoding
      X-Cache:
      - CONFIG_NOCACHE
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - SAMEORIGIN
      X-Msedge-Ref:
      - 'Ref A: 7941D30298214FC48DFBA66767E05C55 Ref B: YTO01EDGE0506 Ref C: 2023-02-21T17:21:20Z'
      X-Tfs-Processid:
      - 05830bec-98a2-44a0-aed1-265f3224303f
      X-Tfs-Session:
      - 133b8869-a401-49b1-8741-3817dc116205
      X-Vss-E2eid:
      - 133b8869-a401-49b1-8741-3817dc116205
      X-Vss-Senderdeploymentid:
      - 4ff21e82-8865-0b2e-ffe8-9598818f8190
      X-Vss-Userdata:
      - 473dec3e-03d7-6147-b106-0b0a7f766a92:idan.varsano@sourcegraph.com
    status: 200 OK
    code: 200
    duration: ""
