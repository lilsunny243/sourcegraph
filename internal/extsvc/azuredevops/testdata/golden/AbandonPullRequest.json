{
  "repository": {
   "id": "c4d186ef-18a6-4de4-a610-aa9ebd4e1faa",
   "name": "sgtestazure",
   "remoteURL": "https://sgtestazure@dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazure",
   "url": "https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa",
   "sshUrl": "git@ssh.dev.azure.com:v3/sgtestazure/sgtestazure/sgtestazure",
   "webUrl": "https://dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazure",
   "isDisabled": false,
   "isFork": false,
   "project": {
    "id": "dc493f7d-0b57-4de2-a59b-3f74ff3ea334",
    "name": "sgtestazure",
    "state": "wellFormed",
    "revision": 11,
    "visibility": "private",
    "url": "https://dev.azure.com/sgtestazure/_apis/projects/dc493f7d-0b57-4de2-a59b-3f74ff3ea334"
   }
  },
  "pullRequestId": 12,
  "codeReviewId": 12,
  "status": "abandoned",
  "creationDate": "2023-02-17T02:21:23.2995071Z",
  "title": "Make it clearer that \"command\" steps are executed on local machine",
  "description": "Make it clearer that \"command\" steps are executed on local machine",
  "createdBy": {
   "id": "473dec3e-03d7-6147-b106-0b0a7f766a92",
   "displayName": "idan.varsano@sourcegraph.com",
   "uniqueName": "idan.varsano@sourcegraph.com",
   "url": "https://spsprodcca1.vssps.visualstudio.com/Ab768986f-7304-43dd-be74-f37e30f66c38/_apis/Identities/473dec3e-03d7-6147-b106-0b0a7f766a92",
   "imageUrl": "https://dev.azure.com/sgtestazure/_api/_common/identityImage?id=473dec3e-03d7-6147-b106-0b0a7f766a92"
  },
  "sourceRefName": "refs/heads/action-docs",
  "targetRefName": "refs/heads/master",
  "mergeStatus": "",
  "mergeId": "168fc5d7-c830-45a0-b8c4-f69e16faa67a",
  "lastMergeSourceCommit": {
   "commitId": "43f23696277d8c412aa5da481009a5ec8302ded4",
   "url": "https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/commits/43f23696277d8c412aa5da481009a5ec8302ded4"
  },
  "lastMergeTargetCommit": {
   "commitId": "12fba599d6b6ba7de2d4c5194cea58090201e4e1",
   "url": "https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/commits/12fba599d6b6ba7de2d4c5194cea58090201e4e1"
  },
  "supportsIterations": true,
  "artifactId": "vstfs:///Git/PullRequestId/dc493f7d-0b57-4de2-a59b-3f74ff3ea334%2fc4d186ef-18a6-4de4-a610-aa9ebd4e1faa%2f12",
  "reviewers": [],
  "forkSource": null,
  "url": "https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/c4d186ef-18a6-4de4-a610-aa9ebd4e1faa/pullRequests/12"
 }